build-system/get-dep-graph.js

```diff
@@ -431,10 +432,11 @@ function unsupportedExtensions(name) {
 exports.singlePassCompile = function(entryModule, options) {
   return exports.getFlags({
     modules: [entryModule].concat(extensions),
-    writeTo: './dist/',
```

Comment:
this will probably have conflicts with my changes here https://github.com/ampproject/amphtml/pull/18393

Reply:
ah cool, so lets get that in and remove this

Reply:
Done